src/layouts/default.html.eco

```diff
@@ -64,7 +64,7 @@
 					<% end %>
 					<li>
 						<a href="//discourse.transformap.co">
-							Forum
+							Agora
```

Comment:
http://discourse.transformap.co/c/community-building > **Communities** it is, then?

@thoka @gandhiano @species